Makefile

```diff
@@ -15,8 +22,8 @@ provision-cargateway: ./fixtures.car
 	car -c ./fixtures.car &
 
 provision-kubo:
-	find ./fixtures -name '*.car' -exec ipfs dag import {} \;
-	find ./fixtures -name '*.ipns-record' -exec sh -c 'ipfs routing put --allow-offline /ipns/$$(basename -s .ipns-record "{}") "{}"' \;
+	find ./fixtures -name '*.car' -exec ipfs dag import --stats --pin-roots=false {} \;
+	find ./fixtures -name '*.ipns-record' -exec sh -c 'ipfs routing put --offline --allow-offline /ipns/$$(basename -s .ipns-record "{}") "{}"' \;
```

Comment:
:thought_balloon:  iiuc this should make `put` finish instantly, and not block on putting them to 20 peers